docs/docs/identity-providers/google.md

```diff
@@ -54,26 +93,34 @@ Then, you'll need to manually open an editor and add an `impersonate_user` field
 ```git
 {
   "type": "service_account",
-  "client_id": "109818058799274859509",
-  ...
-+  "impersonate_user": "user@pomerium.io"
-  ...
+  "project_id": "pomerium-redacted",
+  "private_key_id": "e07f7c93870c7e03f883560ecd8fd0f4d27b0081",
+  "private_key": "-----BEGIN PRIVATE KEY-----\\n-----END PRIVATE KEY-----\n",
+  "client_email": "redacted@pomerium-redacted.iam.gserviceaccount.com",
+  "client_id": "101215990458000334387",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/redacted%40pomerium-redacted.iam.gserviceaccount.com",
++  "impersonate_user": "user@pomerium.com"
 }
 ```
 
-The base64 encoded contents of this public/private key pair json file will used for the value of the `IDP_SERVICE_ACCOUNT` configuration setting.
+The base64 encoded contents of this public/private key pair json file will used for the value of the IdP [service account] configuration setting.
+
+### Set directory permissions for Workspaces
 
-Next we'll delegate G-suite group membership access to the service account we just created .
+Next, we need to give that service account permissionson the GSuite / Workspace side of the house.
 
-1. Go to your G Suite domain's [Admin console](http://admin.google.com/).
-2. Select **Security** from the list of controls. If you don't see Security listed, select More controls 1\. from the gray bar at the bottom of the page, then select Security from the list of controls.
-3. Select **Advanced settings** from the list of options.
-4. Select **Manage API client** access in the Authentication section.
-5. In the **Client name** field enter the service account's **Client ID**.
-6. In the **One or More API Scopes** field enter the following list of scopes: `https://www.googleapis.com/auth/admin.directory.group.readonly` `https://www.googleapis.com/auth/admin.directory.user.readonly`
+1. From your Google Workspace domain's [Admin console](http://admin.google.com/), go to **Main menu** menu**> Security > API controls**.
```

Comment:
formatting looks off in the preview. Maybe a duplicate `menu**`?

Reply:
Thank you, good catch. I think I fixed it.

Reply:
Thank you, good catch. I think I fixed it.